src/Components/ManagePostAccess/ManagePostAccess.jsx

```diff
@@ -2,190 +2,104 @@ import { useEffect, useState } from 'react';
+  const [personHRO, setPersonHRO] = useState(false);
+  const [positionHRO, setPositionHRO] = useState(false);
+
+  const personSelected = selectedPersons.length > 0;
+  const positionSelected = selectedPositions.length > 0;
+  const requirementsMet =
+    (personSelected || positionSelected) && selectedOrgs.length > 0 && selectedRoles.length > 0;
 
   // Filter Options
-  const genericFilters$ = genericFilters?.filters || [];
-  const positionOptions = getGenericFilterOptions(genericFilters$, 'region', 'short_description');
-  const roleOptions = getGenericFilterOptions(genericFilters$, 'skill', 'code');
-  const { data: orgs, loading: orgsLoading } = useDataLoader(api().get, '/fsbid/agenda_employees/reference/current-organizations/');
-  const organizationOptions = (orgs?.data?.length && nameSort(orgs?.data, 'name')) || [];
-
-  // Pagination
-  const [page, setPage] = useState(managePostSelections?.page || 1);
-  const [limit, setLimit] = useState(managePostSelections?.limit || 10);
-  const prevPage = usePrevious(page);
-  const pageSizes = POSITION_MANAGER_PAGE_SIZES;
-
-  const filtersLoading = genericFiltersIsLoading || orgsLoading;
-
-  const getQuery = () => ({
-    'post-access-countries': selectedCountries.map(statusObject => (statusObject?.code)),
-    'post-access-positions': selectedPositions.map(bureauObject => (bureauObject?.code)),
-    'post-access-orgs': selectedOrgs.map(orgObject => (orgObject?.code)),
-    'post-access-persons': selectedPersons.map(gradeObject => (gradeObject?.code)),
-    'post-access-roles': selectedRoles.map(skillObject => (skillObject?.code)),
-    page,
-    limit,
-  });
+  const peopleOptionsHRO = managePostFilters?.personFilters?.filter(
+    person => person.skillCode === '2010' || person.skillCode === '2201') || [];
+  const positionOptionsHRO = managePostFilters?.positionFilters?.filter(
+    position => position.skillCode === '2201' || position.skillCode === '2010') || [];
+  const peopleOptions = personHRO ? peopleOptionsHRO : managePostFilters?.personFilters || [];
+  const positionOptions = positionHRO
+    ? positionOptionsHRO : managePostFilters?.positionFilters || [];
+  const roleOptions = managePostFilters?.roleFilters || [];
+  const organizationOptions = managePostFilters?.orgFilters || [];
+
 
   const getCurrentInputs = () => ({
-    selectedCountries,
     selectedPositions,
     selectedOrgs,
     selectedPersons,
     selectedRoles,
-    page,
-    limit,
   });
 
-  const fetchAndSet = (resetPage = false) => {
-    const filters = [
-      selectedCountries,
-      selectedPositions,
-      selectedOrgs,
-      selectedPersons,
-      selectedRoles,
-    ];
-    if (filters.flat().length === 0) {
-      setClearFilters(false);
-    } else {
-      setClearFilters(true);
-    }
-    if (resetPage) {
-      setPage(1);
-    }
+  const filters = [
+    selectedPositions,
+    selectedOrgs,
+    selectedPersons,
+    selectedRoles,
+  ];
```

Comment:
```suggestion
  const filters = [
    selectedPositions,
    selectedOrgs,
    selectedPersons,
    selectedRoles,
    ...personHRO ? ['Y'] : [],
    ...positionHRO ? ['Y'] : [],
  ];
```
to capture the checkboxes